src/rgw/rgw_redis_queue.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "rgw_redis_common.h"
+
+namespace rgw {
+namespace redisqueue {
+
+using boost::redis::config;
+using boost::redis::connection;
+
+int initQueue(boost::asio::io_context& io, connection* conn, config* cfg,
+              optional_yield y);
+
```

Comment:
please note that there has to be an "init()" and "delete()" functions to the queue

Reply:
Will add those as I explore the requirements of `notify.cc`. 

Have removed the `initQueue` for now.